apollo-router/src/axum_factory/axum_http_server_factory.rs

```diff
@@ -467,7 +467,26 @@ async fn handle_graphql(
         }
         Ok(response) => {
             tracing::info!(counter.apollo_router_session_count_active = -1,);
-            response.response.into_response()
+            let (mut parts, body) = response.response.into_parts();
+
+            //println!("will compress response, accept-encoding == {accept_encoding:?}");
```

Comment:
```suggestion
```

Reply:
fixed in [0fb0ba9](https://github.com/apollographql/router/pull/2986/commits/0fb0ba985404c3652c257bed12044f955b154d53)

Reply:
You're right @bnjjj, i can also see it